ini/native.ini

```diff
@@ -120,11 +120,11 @@ build_unflags   = ${simulator_macos.build_unflags}
 # Simulator for Windows 10
 #
 #  MSYS2 mingw-w64-x86_64 with these packages:
-#  pacman -S --needed base-devel mingw-w64-x86_64-toolchain mingw64/mingw-w64-x86_64-glm mingw64/mingw-w64-x86_64-SDL2 mingw64/mingw-w64-x86_64-SDL2_net
+#  pacman -S --needed base-devel mingw-w64-x86_64-toolchain mingw64/mingw-w64-x86_64-glm mingw64/mingw-w64-x86_64-SDL2 mingw64/mingw-w64-x86_64-SDL2_net mingw-w64-x86_64-dlfcn
 #
 [env:simulator_windows]
 platform    = ${simulator_common.platform}
 extends     = simulator_common
 src_build_flags = ${simulator_common.src_build_flags} -fpermissive
-build_flags = ${simulator_common.build_flags} ${simulator_common.debug_build_flags} -fno-stack-protector -Wl,-subsystem,windows -ldl -lmingw32 -lSDL2main -lSDL2 -lSDL2_net -lopengl32 -lssp
+build_flags = ${simulator_common.build_flags} ${simulator_common.debug_build_flags} -IC:\\msys64\\mingw64\\include\\SDL2 -fno-stack-protector -Wl,-subsystem,windows -ldl -lmingw32 -lSDL2main -lSDL2 -lSDL2_net -lopengl32 -lssp
```

Comment:
if mingw64\include is in default include path, -ISDL2 should be enough...

Reply:
SDL headers get installed into a subdirectory, but generally standard practice when including the header is to omit it, so adding the SDL subdir as an include dir is required (at least for lib code we don't control)

This is usually solved by using the `sdl2-config` program to get the correct build flags, not sure how we can use that on windows from within platformios python scripts.

Reply:
here ive placed them in Marlin\src\HAL\NATIVE_SIM\include\SDL2 but manually

Reply:
It is inconvenient to have to hard-code an include path here, but I suspect this path will work for most people, and is simple to change if you need something different.

It would be better to fix it in a more robust way, but I'd rather not delay the rest of these changes to figure something out.